test/python/v1/rebuild/features/rebuild_history.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Rebuild history
+
+    Background:
+        Given the io-engine is running and a volume is created with more than one replicas
+
+    Scenario: Record Full rebuild of a faulted replica
+        Given a volume with multiple replicas, undergoing IO
+        When one of the replica becomes faulted
+        And as a result, a new replica is hosted on a different pool
+        Then a full rebuild of the new replica happens
```

Comment:
Should we be explicit on when a rebuild event is added? eg:

```suggestion
        Then a full rebuild of the new replica starts
        And when the rebuild completes
```

Reply:
Modified